libnavigation-metrics/src/main/java/com/mapbox/navigation/metrics/internal/TelemetryUtilsDelegate.kt

```diff
@@ -0,0 +1,12 @@
+package com.mapbox.navigation.metrics.internal
+
+import com.mapbox.common.TelemetryUtils
+
+object TelemetryUtilsDelegate {
+
+    fun getEventsCollectionState(): Boolean = TelemetryUtils.getEventsCollectionState()
+
+    fun setEventsCollectionState(enableCollection: Boolean) {
+        TelemetryUtils.setEventsCollectionState(false, {})
```

Comment:
I think you should use `enableCollection` here.

Reply:
good catch